webserver/exceptions.py

```diff
@@ -0,0 +1,13 @@
+class InvalidAPIUsage(Exception):
+    status_code = 400
```

Comment:
I changed status code from optional to required in the constructor (compared to the page I got the snippet from). I missed this. It can be removed